src/app/api/kamu.graphql.interface.ts

```diff
@@ -1,1234 +1,1009 @@
 // THIS FILE IS GENERATED, DO NOT EDIT!
 import { gql } from "@apollo/client/core";
-import { Injectable } from "@angular/core";
-import * as Apollo from "apollo-angular";
+import { Injectable } from '@angular/core';
+import * as Apollo from 'apollo-angular';
 export type Maybe<T> = T | null;
 export type InputMaybe<T> = Maybe<T>;
-export type Exact<T extends { [key: string]: unknown }> = {
-    [K in keyof T]: T[K];
-};
-export type MakeOptional<T, K extends keyof T> = Omit<T, K> & {
-    [SubKey in K]?: Maybe<T[SubKey]>;
-};
-export type MakeMaybe<T, K extends keyof T> = Omit<T, K> & {
-    [SubKey in K]: Maybe<T[SubKey]>;
-};
+export type Exact<T extends { [key: string]: unknown }> = { [K in keyof T]: T[K] };
+export type MakeOptional<T, K extends keyof T> = Omit<T, K> & { [SubKey in K]?: Maybe<T[SubKey]> };
```

Comment:
Please run the code formatter on this file before merging

Reply:
Ok